Obsidian.Docs/docfx.json

```diff
@@ -0,0 +1,71 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "src": "..",
+          "files": [
```

Comment:
I think we'll only want docs for the Obsidian.API namespace. This is anything that would be available to a plugin developer. Internals are not _that_ interesting to document I think. Correct me if I'm wrong.

Reply:
sure, let me change that real quick

Reply:
..actually, we need to leave it like that, since im pretty sure Obsidian.API does use classes and stuff from Obsidian and Obsidian.NBT, so we need to leave it for docs to generate properly

Reply:
I'm fairly sure it only uses NBT, not the core server.

Reply:
alright then

Reply:
nope, only references the source generators.

Reply:
fixed